packages/core/src/test/shared/schemas.test.ts

```diff
@@ -42,32 +42,36 @@ describe('SchemaService', function () {
 
     it('assigns schemas to the yaml extension', async function () {
```

Comment:
when passing some objects through `util.format` I noticed it set a property `_formatted` (perhaps as a cache?), so without these changes the test fails.